libraries/common/subscriptions/helpers/buildRegisterUrl.js

```diff
@@ -0,0 +1,32 @@
+import queryString from 'query-string';
+import {
+  CHECKOUT_PATH,
```

Comment:
Please make this single lined. Its only two variables. Makes it a bit cleaner.

Reply:
done